docs/components/pages/pack-home/index.tsx

```diff
@@ -1,6 +1,5 @@
 import { GradientSectionBorder } from "../home-shared/GradientSectionBorder";
 import { LandingPageGlobalStyles } from "../home-shared/GlobalStyles";
-import { PackBenchmarks } from "./PackBenchmarks";
```

Comment:
want to delete all the unused code from this import too?

Reply:
we might use it again when we add real-world benchmarks